app/javascript/components/sidebar/sectors/actions.js

```diff
@@ -6,9 +6,8 @@ import uniq from 'lodash/uniq';
 import SECTORS_SQL from './sql/sectors.sql';
 
 export const setList = createAction('SECTORS/setList');
-export const setSectorTitles = createAction('SECTORS/setSectorTitles');
 export const setSelectedSector = createAction('SECTORS/setSelectedSector');
-export const setSelectedLayer = createAction('SECTORS/setSelectedLayer');
+export const setSelectedLayers = createAction('SECTORS/setSelectedLayers');
 export const setListLoading = createAction('SECTORS/setListLoading');
```

Comment:
Sort this. Put all setList, setListLoading, setListError together. Then the selected ones